syntax = "proto3";

message Empty {
}

message EdfText {
  string edf = 1;
  string anno = 2;
}

service EdfDatabroker {
  rpc get_next_edf_file(Empty) returns(EdfText);
}
